authorizer-app/src/environments/environment.prod.ts

```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: true,
-  BACKEND_BASE_URL: '/device-auth/backend'
+  BACKEND_BASE_URL: '/rest-sources/backend'
```

Comment:
is this the REST endpoint of the api ?? If yes, perhaps `/rest-sources/api/` ?

Reply:
this is the route served by nginx at the end. For production this paths need to be mentioned before compile time.

Reply:
Ok, Thanks.